consistency/tool/clustering/clustering_test.json

```diff
@@ -21,6 +21,25 @@
             "cleaned-words": []
          }
 
-    }
+    }, 
+    "cluster-words": {
+      "warning-test1": { 
```

Comment:
NIT: Can we rename these as:

"warning-test1" -> "less-than-3-words"
"warning-test2" -> "null-words"